taf/updater/updater_pipeline.py

```diff
@@ -1205,6 +1207,45 @@ def _update_tuf_current_revision():
                     target_filepath,
                     current_commit,
                 )
+
+            # TUF updater does not always check the validity of all metadata files
+            # if timestamp is not updated, the updater will determine that a new version
+            # of the snapshot file does not need to be downloaded and it will not be validated
+            # during the update process, the metadata files that TUF updater downloads is stored
+            # in a separate folder within the temp directory
+            # For each commit, check if the metadata files inside that directory are the same
+            # as the ones in the auth repository's metadata folder at that revision
```

Comment:
This explanation looks like a good candidate to be a docstring. Could we move the code to a function?